server/utils/licenceStatus.js

```diff
@@ -6,7 +6,7 @@ module.exports = {getLicenceStatus};
 
 function getLicenceStatus(licenceRecord) {
 
-    if (allEmpty([licenceRecord, licenceRecord.licence, licenceRecord.status])) {
+    if (!licenceRecord || !licenceRecord.licence || !licenceRecord.status) {
```

Comment:
Why did you decide against allEmpty?

Reply:
I suppose allEmpty is incorrect isn't it, you want 'anyEmpty'

Reply:
if licenceRecord is null, licenceRecord.licence throws an error....